src/renderer/ScreenController.js

```diff
@@ -87,6 +100,7 @@ class Home extends React.Component {
             deltachat={deltachat}
           />
         }
+        { showAbout && (<AboutDialog isOpen={showAbout} onClose={this.onCloseAbout} />) }
```

Comment:
you don't need showAbout && if you have an `isOpen` parameter.

Reply:
More specifically, the overlay div will be part of the dom once it has been rendered:

![screenshot from 2018-11-01 10-47-55](https://user-images.githubusercontent.com/308049/47844808-a4ea9b80-ddc3-11e8-9086-21e87560c226.png)

Lets just remove it. Makes the code cleaner as well.

Reply:
https://github.com/deltachat/deltachat-desktop/commit/a8a4710f15f94c8330dac5d815b51617bd6294e4